Anirveda.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Anirveda</title>
+    <h1> Anirveda</h1>
+    <h2>The TechnoEconomic Club of PDEU</h2>
+</head>
+<body>
+    <img src="anirveda_logo.png.jpg" id="image"
+                alt="Error Loading Image">

```

Comment:
3. Replace the alt text from "Error loading image" to "Anirveda Logo" or "Logo". The purpose of the alt tag is to show what the image is about. If in reason due to network issue or any other issue logo does not appear then this alt text will appear and when you set alt text to "Logo" or "Anirveda Logo" then in absence of the logo it will tell user that here there is logo.